addons/report/views/templates.xml

```diff
@@ -79,6 +79,16 @@
                 <t t-call-assets="web_editor.summernote" t-css="false"/>
                 <t t-call-assets="web_editor.assets_editor" t-css="false"/>
                 <t t-call-assets="report.assets_editor" t-css="false"/>
+                <script>
```

Comment:
got CSRF validation failed on path '/web_editor/attachment/add',

we cannot able to upload local image... without this.